src/collections/books/2020-10-24-the-enterprise-path-to-service-mesh-architectures-2nd-Edition/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "The Enterprise Path to Service Mesh Architectures, 2nd Edition"
```

Comment:
I think it would be much cleaner if we put the `2nd Edition` inside a bracket like this `The Enterprise Path to Service Mesh Architectures (2nd Edition)`